rules-tests/DowngradePhp74/Rector/ClassMethod/DowngradeContravariantArgumentTypeRector/Fixture/skip_phpdoc_method.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Isn't this the exact same? You can probably remove the "expectation" below the -----

Reply:
fixed.